modules/mlops/kubeflow-platform/README.md

```diff
@@ -0,0 +1,79 @@
+## Introduction
+This is an implmententation of [Kubeflow](https://www.kubeflow.org/docs/) via [Kubeflow-on-AWS](https://awslabs.github.io/kubeflow-manifests/docs/).
+It leverages the [Kubeflow-on-AWS github repo](https://github.com/awslabs/kubeflow-manifests) and the [Kubeflow-Manifests Github repo](https://github.com/kubeflow/manifests).
+
+
+
+
+## Description
+
+This is a seedfarmer / ADDF wrapped implementation based off Kubeflow-on-AWS.  This allows customization of the deployment via the module's `deplopyspec`.
+It currently supports only the following:
+- `vanilla` deployment via the `kustomize` deployment method
+- Kubeflow v1.6.1
+- AWSKubebuild 1.0.0
+
+*** NOTE: this module does not support earlier versions of Kubeflow v1.6.1 as specified in the branches of [Kubeflow-on-AWS](https://github.com/awslabs/kubeflow-manifests)
+
+Since this module is extenisble, additions and modifications are encouraged.
+
+
+
+## Prerequisites
+This module depends on an existing EKS cluster and access to EKS Master role (an IAM role that is an admin on the EKS cluster and has IAM capabilty to create/delete roles and policies).
+
+<b>The EKS cluster version MUST be at least 1.23</b>
+
+
+
+## Inputs/Outputs
+
+
+### Input Parameters
+
+
+#### Required
+- `EksClusterMasterRoleArn` - the masterrole in IAM that was used to create the EKS cluster
+- `EksClusterName` - the name of the EKS cluster
+- `InstallationOption` - should be `kustomize` 
+- `DeploymentOption` - please see [Deployment Options](https://awslabs.github.io/kubeflow-manifests/docs/deployment/)
+  - only `vanilla` is currently supported
+- `KubeflowReleaseVersion` - only v1.6.1 currently tested
```

Comment:
As we discussed, v1.6.1 introduced make file which did not exist in the previous versions.
I checked that a naive change of the version to `v1.5.1` ended up with the failure of the deployment. 

It is probably nice to mention that `v1.6.1` must be used here.

Reply:
Already got that one (agreed!)